doc/en/unittest.rst

```diff
@@ -183,7 +183,13 @@ Running this test module ...::
 
     $ pytest -q test_unittest_cleandir.py
     .
-    1 passed in 0.12 seconds
+    ======= warnings summary ========
+    test_unittest_cleandir.py::MyTest::test_method
+      $REGENDOC_TMPDIR/test_unittest_cleandir.py:11: ResourceWarning: unclosed file <_io.TextIOWrapper name='samplefile.ini' mode='r' encoding='UTF-8'>
+        s = open("samplefile.ini").read()
+    
+    -- Docs: http://doc.pytest.org/en/latest/warnings.html
```

Comment:
Oh. We should probably suppress (or fix) warnings in the doc output if irrelevant...